Test/e2e/api/added-to-cart.spec.js

```diff
@@ -54,6 +54,8 @@ test.describe('Added To Cart Event Tracking', () => {
 
         // Validate event data
         expect(eventData.AddedItemProductName).toBe(productName.trim());
+        expect(eventData.external_catalog_id).toBe('1-1');
+        expect(eventData.integration_key).toBe('magento_two');
```

Comment:
These assertions failed for me - assuming the updated extension hasn't been installed on the test site?

Reply:
correct - local playwright tests passed but there are some flaws to my plan to get the branch on the playwright site. gonna work through it today, but these should be good to go